daily_post.py

```diff
@@ -10,29 +12,74 @@
 import daily_markov
 
 
-def get_weather() -> str:
-    resp = requests.get(
-        'http://forecast.weather.gov/MapClick.php?lat=47.62&lon=-122.36&unit=0&lg=english&FcstType=text&TextType=1')
-    doc = resp.content
-    soup_doc = BeautifulSoup(doc, 'html.parser')
-    weather_info = soup_doc.find_all('table')[1]
-    reddit_comment = ''
-    if weather_info.find_all('div'):
-        advisories = weather_info.div.extract().find_all('a')
-        reddit_comment += '* Advisories:\n'
-        for advis in advisories:
-            reddit_comment += ' * [{}](http://forecast.weather.gov/{})\n'.format(
-                advis.span.string, advis.get('href'))
-    forecast = str(weather_info).split("<br/>\n<br/>")
-    top_three = '<br/>\n<br/>'.join(str(x) for x in forecast[1:5])
-    top_three_doc = BeautifulSoup(top_three, 'html.parser')
-    reddit_comment += '* {}'.format(str.replace(
-        top_three_doc.text, '\n\n', "\n* ")[1:-1])
-
-    return reddit_comment
-
-
-def gen_post(weekday: int) -> str:
+DISCOVERY_URL = 'https://api.weather.gov/points/47.62,-122.36'
```

Comment:
Do we need the following?

* DISCOVERY_URL
* FORECAST_GRID_DATA_URL
* FORECAST_HOURLY_URL

Reply:
Removing